tests/inlining_test.py

```diff
@@ -372,6 +415,7 @@ def test(A: dace.float64[96, 32], B: dace.float64[42, 32]):
     # test_regression_reshape_unsqueeze()
     test_empty_memlets()
     test_multistate_inline()
```

Comment:
I'm more worried about a case where an access node is used around the nested SDFG. Please add a test case for the following:
```
  A[0]         B[0]
(A) -> [nsdfg] ----> (B)
    \             / B[1]
     -> [tasklet]   
    A[1]
```
i.e., same output access node, same input access node.

Reply:
According to the algorithm, it would do nothing. It would be important to see if `state_fission_after` will generate an empty state or some orphaned access nodes in this case.